models/rne/dirigeants.tsx

```diff
@@ -33,8 +33,7 @@ export const getDirigeantsRNE = async (
       if (eFallback instanceof HttpNotFound) {
         return APINotRespondingFactory(EAdministration.INPI, 404);
       }
+      return APINotRespondingFactory(EAdministration.INPI, 500);
```

Comment:
Why move this up ?

Reply:
While rereading this I feel it was more understandable like this

Reply:
(there are "return" everywhere except here, so the last return was used only here)